workflows/src/main/resources/migration/allowed-to-see-states-schema.changelog-init.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="R3.Corda" id="create_allowed_to_see_states_table_id">
+        <createTable tableName="account_to_state_refs">
+            <column name="id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="external_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="output_index" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="transaction_id" type="NVARCHAR(64)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="R3.Corda" id="account_to_state_refs_pkey_id">
+        <addPrimaryKey columnNames="id" constraintName="account_to_state_refs_pkey" tableName="account_to_state_refs"/>
+    </changeSet>
+
+    <changeSet author="R3.Corda" id="account_to_state_refs_idx_id">
+        <createIndex indexName="account_to_state_refs_idx" tableName="account_to_state_refs">
```

Comment:
Again, I think the index names will need to match with the JPA entity.

Reply:
Done